orderbook/openapi.yml

```diff
@@ -371,6 +406,18 @@ components:
         signature:
           description: "OrderCancellation signed by owner"
           $ref: "#/components/schemas/Signature"
+    PriceEstimate:
+      description: |
+        Provides the information about an estimated price.
+      type: object
+      properties:
+        price:
+          description: The estimated price
+          type: number
+          format: double
```

Comment:
Can this be a string in decimal format?
Like '105.1234567433' for example

Reply:
What's the advantage? Wouldn't a float be more easily processable?

Reply:
I think this only matters for precision because many json libraries force "numbers" to be f32 or f64. f32 could be a problem if there are tokens with very different number of decimals. So to be safe we might want this to be a string which also allows us to use BigRational in the backend if we want to.

Reply:
Yes, my issue would be that double and floats have some issues with precision. They don't represent the value we actually mean for example:

![image](https://user-images.githubusercontent.com/2352112/111335139-6db77500-8674-11eb-87a8-dcd45cc0e2e7.png)

Reply:
In JS people can use BigNumber libraries, in Python Decimal

Btw, I think https://docs.python.org/3/library/decimal.html library explains great why this model is better. People will also be able to parse these strings as doubles, but the other way around cannot be done without loosing information

Reply:
Makes sense 👍